### PR TITLE
feat: change test case

### PR DESCRIPTION
**Pull request by BhupenMandal**
Changed values for the test case

### CODE REVIEW

**Review by neerajmandal — APPROVED** [review]

File: src/sum.test.js [hunk 1]
```diff
@@ -2,13 +2,13 @@
 const sum = require('./sum')
 
 describe('sum()', () => {
-  test('Given 1 and 2, should sums 3', () => {
+  test('Given 5 and 2, should sum 7', () => {
     const given = {
-      a: 1,
+      a: 5,
```
Comment (neerajmandal) [inline src/sum.test.js:7]: Good job

File: src/sum.test.js [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
       b: 2
     }
 
-    const expected = 3
+    const expected = 7
```
Comment (neerajmandal) [inline src/sum.test.js:11]: Wrong value change this

File: src/sum.test.js (hunk 1 above)
Comment (BhupenMandal) [reply src/sum.test.js:7]: Thanks

File: src/sum.test.js (hunk 2 above)
Comment (BhupenMandal) [reply src/sum.test.js:11]: Change value done

**Pull request merged**